src/main/java/com/commercetools/pspadapter/paymentHandler/impl/PaymentHandler.java

```diff
@@ -135,7 +188,8 @@ public PaymentHandleResponse patchAddress(@Nonnull Cart cart, @Nonnull String pa
     public PaymentHandleResponse executePayment(@Nonnull String paypalPlusPaymentId,
                                                 @Nonnull String paypalPlusPayerId) {
         CompletionStage<PaymentHandleResponse> executeCS = ctpFacade.getCartService()
-                .getByPaymentMethodAndInterfaceId(PaypalPlusPaymentInterfaceName.PAYPAL_PLUS, paypalPlusPaymentId)
+                .getByPaymentMethodAndInterfaceId(PaypalPlusPaymentInterfaceName.PAYPAL_PLUS,
+                        paypalPlusPaymentId, "paymentInfo.payments[*]")
```

Comment:
why not use constants instead of hardcoding such strings?

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/49/commits/f7a43647710700ac50318bce79681e0d1d9179cc